developer/src/kmc/src/kmlmi.ts

```diff
@@ -74,16 +74,14 @@ let modelInfoOptions: ModelInfoOptions = {
   kmpFileName: kmpFilename
 };
 
-try {
-  const data = writeMergedModelMetadataFile(
-    inputFilename,
-    callbacks,
-    modelInfoOptions);
-  fs.writeFileSync(outputFilename, data);
-} catch(e) {
-  console.error(e);
+const data = writeMergedModelMetadataFile(
+  inputFilename,
+  callbacks,
+  modelInfoOptions);
+if(!data) {
   process.exit(SysExits.EX_DATAERR);
 }
+fs.writeFileSync(outputFilename, data);
```

Comment:
So we don't need to handle exceptions we get from `fs.writeFileSync`?

Reply:
kmlmi is deprecated so only doing minimal work on it. Replacement, kmc, has much deeper error management.